### PR TITLE
Update WHSSDatlas_v3.01.jsonld

### PR DESCRIPTION
**Pull request by aeidi89**

### CODE REVIEW

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld [hunk 1]
```diff
@@ -15,8 +15,8 @@
   "copyright": null,
   "custodian": null,
   "description": null,
-  "digitalIdentifier": null,
-  "fullDocumentation": null,
+  "digitalIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017124",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld]: this should be: 
`"digitalIdentifier": {"@id": "https://scicrunch.org/resolver/RRID:SCR_017124"},`

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld [hunk 2]
```diff
@@ -380,20 +380,24 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Osen et al., 2019.",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld]: it is better to to state the full citation. you don't want people to search for the papers (at least state the doi)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld [hunk 3]
```diff
@@ -381,7 +381,7 @@
 		"ontologyIdentifier": null
 	},
   "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Osen et al., 2019.",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the publications for WHS rat brain atlas v1 (Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: 10.1016/j.neuroimage.2014.04.001) and WHS rat brain atlas v3 (Osen KK, Imad J, Wennberg AE, Papp EA, Leergaard TB (2019) Waxholm Space atlas of the rat brain auditory system: Three-dimensional delineations based on structural and diffusion tensor magnetic resonance imaging. NeuroImage 199, 38-56. DOI: 10.1016/j.neuroimage.2019.05.016).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld]: I suggest markdown formatting (cf. example in your other PRs)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld [hunk 4]
```diff
@@ -394,9 +394,9 @@
   "ontologyIdentifier": null,
   "otherContribution": null,
   "relatedPublication": [
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld:396]: please correct indentation (cf. other PRs)

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld (hunk 4 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld:396]: Fixed.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld (hunk 3 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld]: Thanks @lzehl 
Fixed now.

**Review by lzehl — APPROVED** [review]

**Pull request merged**